src/OrchardCore.Components/OrchardCore.Common.Components/OptionEditor.razor.js

```diff
@@ -0,0 +1,10 @@
+export function optionsChanged(element, jsonValue, defaultValue) {
```

Comment:
Could this be renamed to .mjs so that we make a distinction between plain javascript a modules.

Reply:
To trigger the event to DOM

Reply:
Something like @onclick=SetMessage

Reply:
This file using @export suggests that it will become either an CommonJS or EcmaScript module.

Reply:
And that probably means that we would need to convert every small scripts to ES6 modules.